DAST/Script.md

```diff
@@ -21,85 +21,57 @@ Currently you have the option to integrate the SOOS DAST product using the comma
 * In all of our example scripts we are using environment variables, we strongly recommend that you also set up your SOOS_CLIENT_ID and SOOS_API_KEY as environment variables too.
 
 
-### **Script Arguments**
+### Script Arguments
 
-| Name        | Required | Description                                                   |
-|-------------|----------|---------------------------------------------------------------|
-| `targetURL` | Yes      | target URL including the protocol, eg https://www.example.com |
+| Argument | Required | Description |
+| --- | --- | --- |
+| `targetURL` | Yes | Target URL - URL of the site or api to scan. The URL should include the protocol. Ex: https://www.example.com |
```

Comment:
targetURL should mention the "API specification file" when the scan mode is apiscan